job-server/test/spark.jobserver/WebApiSpec.scala

```diff
@@ -42,7 +42,7 @@ with ScalatestRouteTest with HttpService {
   // See http://doc.akka.io/docs/akka/2.2.4/scala/actors.html#Deprecated_Variants;
   // for actors declared as inner classes we need to pass this as first arg
   val dummyActor = system.actorOf(Props(classOf[DummyActor], this))
-  val statusActor = system.actorOf(Props(classOf[JobStatusActor], new InMemoryDAO))
+  val statusActor = system.actorOf(Props(classOf[JobStatusActor], system.actorOf(Props(classOf[JobDAOActor], new InMemoryDAO))))
```

Comment:
Create an `JobStatusActor.props` for it (easier to maintain changes in param lists etc)?